beacon-chain/db/filesystem/blob.go

```diff
@@ -66,6 +56,23 @@ func WithSaveFsync(fsync bool) BlobStorageOption {
 	}
 }
 
+// WithFs allows the afero.Fs implementation to be customized. Used by tests
+// to substitute an in-memory filesystem.
+func WithFs(fs afero.Fs) BlobStorageOption {
+	return func(b *BlobStorage) error {
+		b.fs = fs
+		return nil
+	}
+}
+
+// WithLayout enables the user to specify a runtimeLayout which dictates how blob files are stored on disk.
```

Comment:
The object `runtimeLayout` does not exist in the code base.

Reply:
refactor artifact, will fix!